crates/cheatcodes/src/inspector.rs

```diff
@@ -77,6 +78,53 @@ impl Context {
     }
 }
 
+type AllowCheatcodesFn<DB, CallType> =
+    dyn Fn(&mut Cheatcodes, &mut EvmContext<DB>, &mut CallType) -> Address;
+type CreateOutcomeFn =
+    dyn Fn(InterpreterResult, Option<Address>, Option<Range<usize>>) -> CommonCreateOutcome;
```

Comment:
do we need dyn aliases?

looks like we only use this as function arguments for closures?

perhaps we can enforce this in where clauses

Reply:
sure removed and replaced by FnMut bounds